sigar_windows.go

```diff
@@ -306,6 +312,45 @@ func GetProcName(pid int) (string, error) {
 
 }
 
+func GetProcCredName(pid int) (string, error) {
+	handle, err := syscall.OpenProcess(syscall.PROCESS_QUERY_INFORMATION, false, uint32(pid))
+
+	defer syscall.CloseHandle(handle)
+	
+	if err != nil {
+		return "", fmt.Errorf("OpenProcess fails with %v", err)
+	}
+	
+	var token syscall.Token
```

Comment:
Do you think that is enough reason to use two libraries instead of just one? I'm worried that would confuse things having two libraries hit win32... otherwise we should probably just change ALL the windows method implementations to use golang.org/x/sys/windows instead of syscall.

The other windows gosigar methods use the syscall import to get at win32. What do you think?